modules/paragraphs_bundle_permissions/paragraphs_bundle_permissions.install

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * @file
+ * Install, update and uninstall functions for the paragraphs_bundle_permissions
+ * sub-module.
+ */
+
+/**
+ * Implements hook_install().
+ *
+ * Add default permissions to anonymous role. Admin roles are added by default
+ * when the permissions are defined, so no need to add here.
+ */
+function paragraphs_bundle_permissions_install() {
+  // Load paragraph types/bundles.
+  $bundles = paragraphs_bundle_load();
+  // Create empty array for permissions.
+  $permissions = array();
+  // Define the permissions for each type/bundle.
+  foreach ($bundles as $machine_name => $bundle) {
+    $permission_name = 'view paragraph content ' . $machine_name;
+    $permissions += array(
+      $permission_name => TRUE,
+    );
+  }
+  // Add the permissions to the anonymous role.
+  user_role_change_permissions('anonymous', $permissions);
+
+  // Define a message informing the user default permissions have been added to
+  // each paragraph type.
+  // Get role names.
+  $roles = user_roles();
+  // Compile the message.
+  $t = get_t();
+  $message = $t('Default permissions have been added to the %anonymous role (View) and %administrator role (All) for all existing paragraph types. Review and make changes if needed on the <a href="@url">permissions</a> page.',
+    array(
+      '%anonymous' => $roles['anonymous'],
+      '%administrator' => $roles['administrator'],
+      '@url' => paragraphs_bundle_permissions_url(),
```

Comment:
Can we use the basic permissions page URL and add a query on this link to take advantage of the new permissions filter? 
e.g. build this URL: `admin/config/people/permissions?search=paragraphs`

Then we might not need the special function at all. This link/portion of the message should only show if the user has permission to update permissions.